.circleci/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:18.04
+
+# - "ppa:ubuntu-toolchain-r/test" is used for gcc-9
+# - "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-9 main" is the
+#   official LLVM repository for clang-9
+RUN apt-get update && \
+    apt-get install -y software-properties-common wget && \
+    wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -  && \
+    add-apt-repository ppa:ubuntu-toolchain-r/test && \
+    add-apt-repository "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-9 main"
+
+RUN apt-get update && \
+    apt-get install -y \
+        git \
+        cmake \
```

Comment:
```suggestion
        cmake \
        vim \
```

Reply:
in the case one would need to interact with files on the image

Reply:
sounds good!